charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -444,7 +419,7 @@ ccd:
       memoryRequests: '1524Mi'
       cpuRequests: '900m'
       environment:
-        DATA_STORE_DB_HOST: ${SERVICE_NAME}-postgresql
+        DATA_STORE_DB_HOST: "sscs-flexibleserver-postgres-preview.postgres.database.azure.com"
```

Comment:
you should be setting the global value instead in the global section:
https://github.com/hmcts/sscs-tribunals-case-api/blob/26507b9fd183dc869ae2aa58056ee7daa85b097b/charts/sscs-tribunals-api/values.ccd.preview.template.yaml#L148

removing any references to db host that belong to the ccd chart as that will use the global and in any SSCS dbs you should change to using the global url so it's only referenced once (may need to use tpl actually i might have been wrong)

Reply:
done